lib/stdlib/src/supervisor.erl

```diff
@@ -132,16 +132,16 @@
 -type child_rec() :: #child{}.
 
 -record(state, {name,
-		strategy               :: strategy() | 'undefined',
+		strategy = one_for_one:: strategy(),
 		children = {[],#{}}    :: children(), % Ids in start order
-                dynamics               :: {'maps', #{pid() => list()}}
-                                        | {'mapsets', #{pid() => []}}
-                                        | 'undefined',
-		intensity              :: non_neg_integer() | 'undefined',
-		period                 :: pos_integer() | 'undefined',
+        dynamics               :: {'maps', #{pid() => list()}}
+                                | {'mapsets', #{pid() => []}}
+                                | 'undefined',
```

Comment:
```suggestion
                dynamics               :: {'maps', #{pid() => list()}}
                                        | {'mapsets', #{pid() => []}}
                                        | 'undefined',
```
This did not change anything except for indentation which I think was correct before